packages/app/obojobo-repository/setupTests.js

```diff
@@ -0,0 +1 @@
+require('obojobo-lib-utils/test-setup-chunks') // setup enzyme
```

Comment:
just making sure this is needed (if so maybe we should rename test-setup-chunks)

Reply:
Not sure about the name, but some of the tests use Enzyme so this'll definitely be necessary.